src/library/Uno.Material.WinUI.Markup/Uno.Material.WinUI.Markup.csproj

```diff
@@ -19,7 +19,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Uno.Extensions.Markup.Generators" PrivateAssets="All" Version="5.0.0-dev.407" />
+		<PackageReference Include="Uno.Extensions.Markup.Generators" PrivateAssets="All" Version="5.0.0-dev.448" />
```

Comment:
Should we use/test with latest [5.0.0-dev.517](https://www.nuget.org/packages/Uno.Extensions.Markup.Generators/5.0.0-dev.517) ?

Reply:
not necessary, in fact we dont even need to update anymore
the bump was to grab the fix that make the attribute applicable to fields

Reply:
if you're using the Generators I would advise that you keep that update to date.